.github/workflows/build.yml

```diff
@@ -27,13 +27,8 @@ jobs:
     - name: Setup Python
       uses: actions/setup-python@v2
 
-    - name: Setup Node.js
-      uses: actions/setup-node@v1
-
     - name: Checkout code
       uses: actions/checkout@v2
-      with:
-        submodules: 'true'
```

Comment:
Oh, submodules are not required for this?  I will defer to you, just want to double-check.

Reply:
yeah, sub-modules basically is the tinyusb and other mcu driver, the native doesn't uses any of those. ci passed is the proof for that. Those mcu is actually very large for some such as nxp and esp-idf (both > 100MB)